src/components/step/ConnectEhrStep/ConnectEhrStep.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { ProviderSearch } from "../../container";
+import StepLayout from "../StepLayout";
+import StepNextButton from "../StepNextButton";
+import StepText from "../StepText";
+import StepTitle from "../StepTitle";
+import MyDataHelps, { ExternalAccountProvider } from "@careevolution/mydatahelps-js";
+
+export interface ConnectEhrStepProps {
+    title?: string;
+    text?: string;
+    nextButtonText?: string;
+    nextButtonDisabled?: boolean;
+    previewState?: boolean;
+    onProviderSelected: (provider: ExternalAccountProvider) => void;
+    styles: { [key: string]: any };
+}
+
+export default function (props: ConnectEhrStepProps) {
+    return (
+        <StepLayout>
+            <StepTitle
+                text={props.title}
+                textAlign={props.styles.titleAlignment}
+                color={props.styles.titleColor}
+                fontSize={props.styles.titleFontSize}
+                fontWeight={props.styles.titleFontWeight}
+            />
+            <StepText
+                text={props.text}
+                textAlign={props.styles.textAlignment}
+                color={props.styles.textColor}
+                fontSize={props.styles.textFontSize}
+                fontWeight={props.styles.textFontWeight}
+            />
+            <ProviderSearch
+                previewState={props.previewState ? "Default" : undefined}
```

Comment:
Could you add some doc around this previewState property? What is it doing and what are the allowed values. Is it just whether the search is executed to preview the list of providers on component load?

Reply:
It allows Storybook to display ConnectEhrStep with the default provider search preview state. Would it be clearer if we use the type of `ProviderSearchPreviewState` instead?

Reply:
Yes, I think that would be clearer naming. But do you mean this property is only used because of the storybook? A comment explaining the use and reason would be good.

Reply:
Yes, it only exists because of the storybook. I will add a comment.

Reply:
Anything else?